tests/cases/fourslash/commentsFunction.ts

```diff
@@ -33,6 +33,20 @@
 ////    }
 ////    return lamb/*31*/daVar("World") + /*32*/a;
 ////}
+/////**
+//// * On variable
+//// * @param s the first parameter!
+//// * @returns the parameter's length
+//// */
+////var assi/*33*/gned = /**
+////                * Summary on expression
+////                * @param s param on expression
+////                * @returns return on expression
+////                */function(/** On parameter */s: string) {
+////  return s.length;
+////}
+////assig/*34*/ned/*35*/(/*36*/"hey");
+

```

Comment:
done. I split it in half -- one for testing function declarations and one for function expressions. There's still a lot of redundancy in there, but testing `=>` vs `function` *might* someday expose a bug. I guess.